CHANGES.rst

```diff
@@ -1,6 +1,31 @@
 2.0.0
 =====
 
+Summary
+-------
+
+Many thanks to all of our awesome contributors (listed down below) who made
+this release possible!
+
+In 2.0 we added a number of new convenience methods and properties. We also
+made it a lot cleaner and less-error-prone to assign multiple routes to the
+same resource class via suffixed responders. Middleware methods can now
+short-circuit request processing. Also, we improved cookie and ETag handling.
+The testing framework also received several improvements to make it easier
```

Comment:
Awesome summary!

Just a minor nitpick, word *also* is used in two sentences in a row here, and thrice in this paragraph.

Reply:
Took out one of them, should read better now.